### PR TITLE
97  create an employee card

### PR DESCRIPTION
**Pull request by ASGladchenko**

### CODE REVIEW

File: client/src/components/dashboard/dashboard-header/index.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const DashboardHeader = () => {
   };
 
   return (
-    <header className="w-full bg-boardHeader px-[32px] desktopXl:px-0">
+    <header className="w-full bg-boardHeader px-[32px] desktopXl:px-0 ">
```
Comment (Wolshebnik) [inline client/src/components/dashboard/dashboard-header/index.tsx:21]: remove space

File: client/src/components/employee-card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+'use client';
+import { useRouter } from 'next/navigation';
```
Comment (Wolshebnik) [inline client/src/components/employee-card/index.tsx:2]: space

File: client/src/components/employee-card/job-title/index.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+'use client';
```
Comment (Wolshebnik) [inline client/src/components/employee-card/job-title/index.tsx:1]: space

File: client/src/components/employee-card/types.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export enum Status {
+  Active = 'Active',
+  Blocked = 'Blocked',
+}
```
Comment (Wolshebnik) [inline client/src/components/employee-card/types.ts:4]: space

File: client/src/constants/routes.ts [hunk 5]
```diff
@@ -9,4 +9,5 @@ export const routes = {
   declined: '/admin/declined',
   aboutService: '/about_service',
   organizations: '/admin/organizations',
+  employees: '/manager/employees',
```
Comment (Wolshebnik) [inline client/src/constants/routes.ts:12]: sort

**Review by Wolshebnik — APPROVED** [review]

**Review by Kyslytsia — APPROVED** [review]

**Pull request merged**